requirements-dev-only-frozen.txt

```diff
@@ -12,7 +12,7 @@ distlib==0.3.4
 docker==3.7.0
 docker-pycreds==0.4.0
 exam==0.5.1
-filelock==3.7.0
+filelock==3.7.1
```

Comment:
What's this for?

Reply:
Oh, I see. Assuming you undo this.

Reply:
it's only used in pre-commit (via `virtualenv`) so I'm just going to bump it anyway